include/ur_client_library/control/script_command_interface.h

```diff
@@ -0,0 +1,114 @@
+// this is for emacs file handling -*- mode: c++; indent-tabs-mode: nil -*-
+
+// -- BEGIN LICENSE BLOCK ----------------------------------------------
+// Copyright 2021 FZI Forschungszentrum Informatik
+// Created on behalf of Universal Robots A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -- END LICENSE BLOCK ------------------------------------------------
+
+//----------------------------------------------------------------------
+/*!\file
+ *
+ * \author  Mads Holm Peters mahp@universal-robots.com
+ * \date    2022-08-12
+ *
+ */
+//----------------------------------------------------------------------
+
+#ifndef UR_CLIENT_LIBRARY_SCRIPT_COMMAND_INTERFACE_H_INCLUDED
+#define UR_CLIENT_LIBRARY_SCRIPT_COMMAND_INTERFACE_H_INCLUDED
+
+#include "ur_client_library/control/reverse_interface.h"
+#include "ur_client_library/ur/tool_communication.h"
+
+namespace urcl
+{
+namespace control
+{
+/*!
+ * \brief Available script commands
+ */
+enum class ScriptCommand : int32_t
+{
+
+  ZERO_FTSENSOR = 0,     ///< Zero force torque sensor
+  SET_PAYLOAD = 1,       ///< Set payload
+  SET_TOOL_VOLTAGE = 2,  ///< Set tool voltage
+};
```

Comment:
Is there a certain intention why this enum is not a private class member? It seems like this is only used internally and not necessarily meant to be used externally, since you wrapped the different commands into individual functions.

Reply:
No, I think I just copied the setup from one of the other classes. Since it is only used inside the class it definitely makes sense to make it private.